docs/hugo.yaml

```diff
@@ -1,4 +1,4 @@
-baseURL: /
+baseURL: https://distribution.github.io/distribution
```

Comment:
Hmm, this is interesting. What does this change make? Is it to make the GH changes work with the new hooks?

Reply:
No, it's actually how this should've been.  Think of this as the intended absolute url for the deploy; in local deploy it overrides for dev reasons, but this is supposed to be the end host + url.